test/e2e/specs/customer_admin.go

```diff
@@ -240,5 +241,26 @@ var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin][Fa
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	It("should sync AAD admin group", func() {
+		syncSecret, err := cli.Client.Admin.CoreV1.Secrets("openshift-infra").Get("aad-group-sync-config", metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		idPre := "44e69b"
```

Comment:
I don't believe it's an excessive risk to check the full guid or full username in.

Reply:
I agree that the guid is probably fine.  I'd still prefer not to expose the account information.